test/unit/InputObserver.spec.js

```diff
@@ -105,6 +105,42 @@ describe("InputObserver", function () {
         this.inst.change(inputType, {}, { y: sign * 40 });
         this.inst.release(inputType);
       });
+
+      it(`should check delta that dragged bounce area (direction: ${direction})`, done => {
```

Comment:
@daybrush What is the difference with above test `should check delta that dragged out of bounce area(direction: ${direction})`?

Reply:
@jongmoon out of bounce area is really outside except bounce.